SpiceSharp/Components/RLC/CAP/TemperatureBehavior.cs

```diff
@@ -98,15 +106,25 @@ public void Temperature(BaseSimulation simulation)
                 var width = BaseParameters.Width.Given
                     ? BaseParameters.Width.Value
                     : ModelParameters.DefaultWidth.Value;
-                Capacitance = ModelParameters.JunctionCap *
+                capacitance = ModelParameters.JunctionCap *
                               (width - ModelParameters.Narrow) *
                               (BaseParameters.Length - ModelParameters.Narrow) +
                               ModelParameters.JunctionCapSidewall * 2 * (
                                   BaseParameters.Length - ModelParameters.Narrow +
                                   (width - ModelParameters.Narrow));
             }
             else
-                Capacitance = BaseParameters.Capacitance;
+                capacitance = BaseParameters.Capacitance;
+
+            double difference = BaseParameters.Temperature - (ModelParameters?.NominalTemperature.Value ?? Circuit.ReferenceTemperature);
```

Comment:
This line is only necessary if `ModelParameters != null`, should be moved inside the check.